toolchain/check/facet_type.cpp

```diff
@@ -107,17 +113,26 @@ auto InitialFacetTypeImplWitness(
   SemIR::InstId witness_inst_id = SemIR::InstId::None;
   llvm::MutableArrayRef<SemIR::InstId> table;
   {
-    llvm::SmallVector<SemIR::InstId> empty_table(
-        assoc_entities.size(),
-        SemIR::ImplWitnessTablePlaceholder::SingletonInstId);
-    auto table_id = context.inst_blocks().Add(empty_table);
-    table = context.inst_blocks().GetMutable(table_id);
+    auto elements_id =
+        context.inst_blocks().Add(llvm::SmallVector<SemIR::InstId>(
+            assoc_entities.size(),
+            SemIR::ImplWitnessTablePlaceholder::SingletonInstId));
```

Comment:
Maybe switch to `AddUninitialized` and then set the values? Allocating a `SmallVector` just to copy it is probably higher overhead

Reply:
Done